its/plugin/tests/src/test/java/com/sonar/it/java/suite/JavaTestSuite.java

```diff
@@ -52,12 +52,11 @@
   SuppressWarningTest.class,
   SonarLintTest.class,
   ExternalReportTest.class,
-  DuplicationTest.class,
-  JspTest.class
```

Comment:
ohlala, nop, we just dropped a test here.

Reply:
If you look at the log of the QA, you will see `com.sonar.it.java.JspTest`.
I move JspTest out of JavaTestSuite because it requires a different version of SQ.
But it is still executed because of the pom.xml `<include>com/sonar/it/java/*Test.java</include>`